### PR TITLE
Feature/rate limit token form

### PR DESCRIPTION
**Pull request by davisnando**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
     'django-otp>=0.5.2',
     'six>=1.1',
     'qrcode>=6.1',
+    'django-ratelimit @ git+https://github.com/jsocol/django-ratelimit.git@c946c076acf2e62374114242620d9170612f83d7#egg=django-ratelimit',
```
Comment (mvantellingen) [inline setup.py]: I don't think we want to release this package with this line

**Comment by mikedingjan** [comment]
@davisnando is it possible to use a released version of the package, so we can merge this merge-request?

**Comment by davisnando** [comment]
@mikedingjan The package rate-limit just needs to release 3.0. 3.0 is already in the changelog but has not been released yet

**Comment by MichielBijland** [comment]
This has become obsolete as 0.6.0 or higher from django-otp has it build in.

**Pull request closed without merging**